Lib/test/test_codecs.py

```diff
@@ -3439,6 +3439,19 @@ def search_function(encoding):
         self.assertEqual(NOT_FOUND, codecs.lookup('BBB.8'))
         self.assertEqual(NOT_FOUND, codecs.lookup('a\xe9\u20ac-8'))
 
+    def test_encodings_normalize_encoding(self):
```

Comment:
nitpick: I suggest to use:

```
normalize = encodings.normalize_encoding
```

and then:

```
self.assertEqual(normalize('utf_8'), 'utf_8') 
```

Just to make the test easier to read.

Reply:
Thanks, done.